book/layout.md

```diff
@@ -654,29 +651,32 @@ commands in one place.
 
 ``` {.python}
 class BlockLayout:
-    def paint(self, display_list):
-        for x, y, word, font in self.display_list:
-            display_list.append(DrawText(x, y, word, font))
-        # ...
+    def paint(self):
+        cmds = []
+        if self.layout_mode() == "inline":
```

Comment:
Why do we need the test here? If it's not in inline mode, the `display_list` should be empty.

Reply:
This code was there before my CL. If it needs to change it should be in a new PR.